addons/account_sepa/account_payment.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api, _
+
+class account_payment(models.Model):
+    _inherit = "account.payment"
+
+    @api.one
+    @api.depends('payment_method')
+    def _compute_sepa_payment_method_selected(self):
+        self.sepa_payment_method_selected = self.payment_method.code == 'sepa_ct'
+
+    sepa_payment_method_selected = fields.Boolean(compute='_compute_sepa_payment_method_selected')
+    sepa_communication = fields.Char(string="Communication", size=140)
```

Comment:
i wonder if this field shouldn't belong to accont module (name= communication) because you also probably need it for the check writing too